src/index.js

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import Calculator from './components/calculator';
-import './components/styles.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Calculator from './pages/calculator';
+import NavLink from './components/nav';
+import './modules-css/calculator.css';
+import './modules-css/nav.css';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <Calculator />
+    <>
+      <BrowserRouter>
+        <div className="header">
+          <NavLink />
+        </div>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quote" element={<Quote />} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
- [ OPTIONAL ] I highly suggest adding the routes inside your app.js to make your code follow the best react flow and the index.js just keep the app there 💯💪🏻🔥🔥